# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../crm114/make_install.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2008-08-13 10:11:56.000000000 -0400
+++ ./Makefile	2008-08-13 10:17:18.000000000 -0400
@@ -7,6 +7,8 @@
 #
 prefix?=/usr
 BINDIR=${prefix}/bin
+DOCDIR=${prefix}/share/doc/crm114
+DATADIR=${prefix}/share/crm114
 
 # VER_SUFFIX defines a version suffix for our installed executables, 
 # handy when you want many versions of CRM114 coexisting.
@@ -474,6 +476,11 @@
 	install $(INSTALLFLAGS) osbf-util $(BINDIR)/osbf-util$(VER_SUFFIX)
 	-install $(INSTALLFLAGS) crm114-mode.el $(prefix)/share/emacs/site-lisp/crm114-mode.el$(VER_SUFFIX)
 
+install_files:
+	mkdir -p $(DOCDIR) $(DATADIR)
+	cp -v $(CRMFILES) $(DATADIR)
+	cp -v $(TEXTFILES) $(DOCDIR)
+
 uninstall: FORCE
 	-rm -rf $(BINDIR)/crm$(VER_SUFFIX)
 	-rm -rf $(BINDIR)/crmg$(VER_SUFFIX)
